marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/hub-entities.xqy

```diff
@@ -523,3 +525,42 @@ declare function hent:json-schema-generate($entity-title as xs:string, $uber-mod
   )
   return xdmp:to-json($uber-model)
 };
+
+declare %private function hent:add-range-indexes-to-model($entities as json:array) {
```

Comment:
I think this function deserves a comment, because for a new developer, it begs the question - Wait, aren't range indexes already supported in the model?

We want to make it immediately clear that the intent here is to support the custom DHF properties of facetable (and eventually sortable) by adding range indexes based on their existence.

Reply:
@rjrudin I will add the function level comment